src/ReactImageMagnify.js

```diff
@@ -123,13 +141,15 @@ ReactImageMagnify.propTypes = {
     largeImage: ImageShape,
     lensStyle: PropTypes.object,
     smallImage: ImageShape,
-    style: PropTypes.object
+    style: PropTypes.object,
+    enlargedImagePosition: PropTypes.string
```

Comment:
What do you think about changing the prop validation to `React.PropTypes.oneOf(['beside', 'over'])`? I'm thinking It helps to document the API and it might give valuable feedback in the scenario where a developer has a typo or uses the wrong word by mistake. If using `oneOf` breaks the test for prop types (`it has correct prop types`), then I think I am okay with removing the test, or finding a different (better) way to test prop types. The current test for prop types is not supported by React and is not a standard method in the community. Please let me know your thoughts!